components/Internship/CommitteeInternshipInfo.js

```diff
@@ -0,0 +1,67 @@
+import Image from 'next/image';
+import React, {useState} from 'react';
+
+function CommitteeInternshipInfo(props) {
+    const committees = props.committees;
+    const [committeeName, setCommitteeName] = useState('Board');
+    return(
+        <div>
```

Comment:
Is there a reason why there's this parent div? Worried about having too many divs haha, esp wrt accessibility tree.

Reply:
there's an error without this since not everything is within a single div! the div right below this is for the line of logos.

Reply:
In that case, you should use a [Fragment](https://reactjs.org/docs/fragments.html) - the empty tag `<> </>` should be fine for this use-case.